<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_pw">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 20px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Passward</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 20px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Login">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Join">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STARBUCKS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_pw</tabstop>
  <tabstop>pb_Login</tabstop>
  <tabstop>pb_Join</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
